hibernate-core/src/test/java/org/hibernate/test/jpa/procedure/JpaTckUsageTest.java

```diff
@@ -46,11 +45,17 @@
  *
  * @author Steve Ebersole
  */
-public class JpaTckUsageTest extends BaseUnitTestCase {
+@RequiresDialect(DerbyTenSevenDialect.class)
```

Comment:
AFAICT this can't ever be executed because we don't instantiate these dialect subclasses anymore :)